views/index.pug

```diff
@@ -32,7 +32,7 @@ html
             //-    .indexPlanetImg
             //-    p Planet
       .indexWrapperDiv#videoDiv
-         iframe(src="https://www.youtube.com/embed/DZg2zUqIdCI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen)
+         iframe(style="width: 85vw; height: 80vh;" src="https://www.youtube.com/embed/DZg2zUqIdCI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen)
```

Comment:
Since this changes the height/width, we really need to have screen shots in order to see how this affects the design.

Reply:
for the size font
![image](https://user-images.githubusercontent.com/48721623/115378052-7863ad80-a1d0-11eb-86af-e8c88757789f.png)

for the video 
![image](https://user-images.githubusercontent.com/48721623/115381814-3b99b580-a1d4-11eb-8de8-54a1ce4490b0.png)

![image](https://user-images.githubusercontent.com/48721623/115381940-6552dc80-a1d4-11eb-8a39-9a7bebcb3dbd.png)